pages/your_tasks/data_discoverability.md

```diff
@@ -0,0 +1,79 @@
+---
+title: Data Discoverability
+description: How to make data discoverable
+contributors: [Aina Jené Cortada, Laura Portell Silva]
+page_id: data_discoverability
+related_pages: 
+  your_task: sensitive
+---
+
+## How can you make your data more discoverable?
+ 
+### Description
+
+Data discovery involves processes and tools that help users understand what data is available, where it is stored, and how to access it. It includes querying datasets to find specific information based on given conditions. However, for data to be discoverable, it must be well-prepared. Making your data discoverable maximizes its impact and utility, enabling others to find, access, and use it effectively. Discoverable data promotes transparency, reproducibility, and scientific progress. Achieving this requires detailed metadata and documentation, depositing data in public and institutional repositories, and using standardized formats for interoperability.
+
+### Considerations
+
+* Detailed Metadata: Provide comprehensive metadata for your datasets, including titles, descriptions, keywords, creators, dates, and other relevant information.
+* Documentation: Include thorough documentation that explains the dataset, its structure, how it was collected, processed, and any limitations.
+* Standard Schemas: Use standardized metadata schemas to ensure consistency and interoperability.
+* Standard Formats: Use widely accepted data formats to ensure compatibility and ease of use.
+* Public Repositories: Deposit your data in reputable public data repositories that are indexed by search engines and widely used by the research community.
+
+### Solutions
+
+* There are many appropriate tools for detailed metadata and data documentation during the project. Check the [Documentation and metadata page](metadata_management) for more information.
```

Comment:
You mean "There are several appropriate tools to create detailed (or comprehensive) metadata and document data properly for the project (not just during?) - check out the [Documentation...."

Reply:
changed the text a bit to clarify